purchases/src/test/java/com/revenuecat/purchases/utils/LocaleExtensionsTest.kt

```diff
@@ -70,4 +78,11 @@ class LocaleExtensionsTest {
         val otherLocale = Locale("zh-Hans")
         assertThat(locale.sharedLanguageCodeWith(otherLocale)).isFalse
     }
+
+    @Test
+    fun `getDefaultLocales returns the correct list`() {
+        assertThat(
+            getDefaultLocales().map { it.toString() }
+        ).isEqualTo(listOf("en_US"))
```

Comment:
Hmm if I'm not wrong, this test will only pass on devices where the default locale is `en_US`?

Reply:
True. Do you have any suggestions that wouldn't lead to false positives (like overriding the locale ourselves).

Reply:
Hmm difficult to do without mocking, which would lose the purpose of the test... Honestly the value of the test is limited since the unit tests don't run in an android image but on the JVM virtual machine (in some cases, there are some differences)

Having said that, I think we can either keep it as is, even if it might fail in some devices, or just return early if the locale is not `Locale.US` so we don't test in those devices. Feels like the first option might be safer but we should add a comment so when it fails, we are aware of this

Reply:
Yeah ideally we only run this on CI, where I think it's safe to assume it's that locale. But we don't have a way to detect that, right?

Reply:
Not right now no... We could potentially pass a parameter in CI for that through gradle. That could be an option as well yeah

Reply:
Let me add a comment for now then.